### DISCUSSION TITLE
alt-i/h toggles doesn't work on MacOS

### DISCUSSION THREAD

**Discussion by Jomik** [ibhagwan/fzf-lua#1331] [Q&A] [answered]
Hey,
I just switched to fzf.lua, and saw that I can use `alt+h` to toggle hidden files, which I definitely need to search into `.github/` and such.
But `alt-h` on my MacOS simply outputs `˙`, which is not what we need.
Can we change this shortcut to be `ctrl+h` somehow? I couldn't immediately see where this is done.

Thanks!

**Comment by ibhagwan** [comment]
There is no default bind for `alt-h`, the only default we have is `ctrl-g` to toggle no ignore:
https://github.com/ibhagwan/fzf-lua/blob/3d214f8db4a4119723ab5148b7a333aab3fa9063/lua/fzf-lua/defaults.lua#L319

If you wish to add `ctrl-h` for hidden, add the below actions to your `setup.files`:
```lua
require 'fzf-lua' {
  files = {
    actions = { ["ctrl-h"] = { actions.toggle_hidden } },
  }
}
```

**Comment by Jomik** [comment, marked as answer]
Oh, jeez. You are right of course. I see now that the actions are set by LazyVim.
My bad, and thank you! Easy fix 👍
